### DISCUSSION TITLE
Remove "Code 55% faster with GitHub Copilot" advertisement from repositories

### DISCUSSION THREAD

**Discussion by guest271314** [community/community#65856] [Copilot Conversations]
### Select Topic Area

General

### Body

There is already a "Try Copilot" advertisement in the drop-down for user account.

I know you are selling that product. I'm not interested and don't need to see your product advertised on the panel of the file viewer in reprositories. Thanks.

![Screenshot_2023-09-03_18-18-06](https://github.com/community/community/assets/4174848/b6586a4e-987a-4eaf-bd7a-dae8c1d3859f)

**Comment by Akash1134** [comment]
Hi @guest271314 ,

Thank you for sharing the feedback. We have passes it onto the relevant team. Similar feedback has also been shared [in this discussion.](https://github.com/orgs/community/discussions/65073)

**Comment by davecheney** [comment]
Duplicate of https://github.com/orgs/community/discussions/65073

**Comment by draeath** [comment]
If you use uBlock Origin, the following filter rule will remove this nonsense (at this time):

    github.com##[data-testid="copilot-popover-button"]

**Reply by cschreib** [reply]
Works with Adblock Plus too, thanks.

**Reply by Protuhj** [reply]
That no longer worked for me, this one worked though: 
`github.com##div.react-code-size-details-in-header:if(span:has-text(Code 55% faster with GitHub Copilot))` - this won't block the other div with the same class.

**Reply by guest271314** [reply]
@Protuhj This https://github.com/guest271314/no-copilot-ad/blob/main/no-copilot-ad.css still works AFAICT

```
.react-code-size-details-in-header:nth-of-type(2),[data-testid="copilot-popover-button"]{display:none!important}
```

**Reply by Protuhj** [reply]
Nice, that looks like it would work too.
